pkg/go.mod

```diff
@@ -37,7 +37,6 @@ require (
 	github.com/opentracing/opentracing-go v1.2.0
 	github.com/pgavlin/goldmark v1.1.33-0.20200616210433-b5eb04559386
 	github.com/pulumi/pulumi/sdk/v3 v3.38.0
-	github.com/rjeczalik/notify v0.9.2
```

Comment:
🎉  any left over CGO stuff we can clean up as part of removing this?

Reply:
This is the last piece - I've got a follow up PR we can discuss this week that switches our build jobs to Linux only and removes CGO_ENABLED=1 and it did pass checks.

Reply:
PR here, though it won't pass checks until this PR is in:
- https://github.com/pulumi/pulumi/pull/10427